docs/getting-started/cli-getting-started.md

```diff
@@ -24,43 +47,85 @@ To install the core Zowe CLI:
 npm install -g @zowe/cli@zowe-v3-lts
 ```
 
-### Installing CLI plug-ins
+If an `EACCESS` error displays and you are running MacOS or Linux systems, there are workarounds available. See the instructions for using the `sudo` command in [Installing Zowe CLI and Zowe CLI plug-ins](../user-guide/cli-installcli).
+
+### Installing Zowe CLI plug-ins
 
 To install Zowe CLI plug-ins:
 
 ```
 zowe plugins install @zowe/cics-for-zowe-cli@zowe-v3-lts @zowe/db2-for-zowe-cli@zowe-v3-lts @zowe/mq-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3-lts
 ```
 
-The preceding command installs most open-source plug-ins, but the IBM Db2 plug-in requires [additional configuration to install](../user-guide/cli-db2plugin.md#installing).
+The preceding command installs [IBM® CICS® Plug-in for Zowe CLI](../user-guide/cli-cicsplugin), [IBM® Db2® Plug-in for Zowe CLI](../user-guide/cli-db2plugin), [IBM z/OS FTP Plug-in for Zowe CLI](../user-guide/cli-ftpplugin), and [IBM MQ Plug-in for Zowe CLI](../user-guide/cli-mqplugin). Remove a plug-in from the command to not install it.
```

Comment:
Can we list the plug-ins in the same order as the install command? Meaning MQ before FTP